hwbench/environment/turbostat.py

```diff
@@ -65,10 +68,12 @@ def __init__(
         self.header = ""
         self.freq_metrics = freq_metrics
         self.power_metrics = power_metrics
+        self.ipc_metrics = ipc_metrics
         self.hardware = hardware
         self.process: subprocess.Popen[bytes] = None  # type: ignore[assignment]
         self.freq_metrics[str(CPUContext.CPU)] = {}  # type: ignore[no-redef]
         self.power_metrics[str(PowerContext.CPU)] = {}  # type: ignore[no-redef]
+        self.ipc_metrics[str(PowerContext.CPU)] = {}  # type: ignore[no-redef]
```

Comment:
Why do we use PowerContext instead of CPUContext here? Later in the code we use CPUContext for ipc_metrics.